mongo/mongo.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2021 Flamego. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package mongo
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/flamego/session"
+)
+
+var _ session.Store = (*mongoStore)(nil)
+
+// mongoStore is a MongoDB implementation of the session store.
+type mongoStore struct {
+	nowFunc    func() time.Time // The function to return the current time
+	lifetime   time.Duration    // The duration to have no access to a session before being recycled
```

Comment:
```suggestion
	lifetime   time.Duration    // The duration to have access to a session before being recycled
```

This looks like a typo in the original copy, could you also update the comment in `mysql/mysql.go` and `postgres/postgres.go`?

Reply:
Sure.